cellpack/tests/test_comp_doc.py

```diff
@@ -11,13 +11,15 @@ def test_composition_doc_as_dict():
         object={"test_key": "test_value"},
         regions={},
         molarity=None,
+        priority=None,
```

Comment:
nit: should we put a value in for priority for one of these tests? instead of just None?